cmd/fsck.go

```diff
@@ -64,6 +64,10 @@ $ juicefs fsck redis://localhost --path /d1/d2 --recursive`,
 				Aliases: []string{"r"},
 				Usage:   "recursively check or repair",
 			},
+			&cli.BoolFlag{
+				Name:  "stat-all",
```

Comment:
need a more understandable flag name/usage.

Reply:
Renamed to `sync-dir-stat`